src/sarif_converter.ts

```diff
@@ -67,10 +70,11 @@ function getRulesAndAllIssues(analysisResults: IAnalysisResult, allIssuesBySugge
     const language = suggestion.id.split('%2F')[0];
     const suggestionId = `${language}/${suggestion.rule}`;
     const ruleProperties: RuleProperties = {
-      tags: [language, ...suggestion.tags, ...suggestion.categories],
+      tags: [language, ...suggestion.tags],
+      categories: suggestion.categories,
```

Comment:
@ork what about:
```suggestion
      tags: [language, ...suggestion.tags, ...categories],
      categories: suggestion.categories,
```

Reply:
I mean, `categories` is not part of Sarif....

Reply:
yes but nor is exampleCommitFixes, exampleCommitDescriptions, precision.